libs/blocks/quiz-results/quiz-results.css

```diff
@@ -0,0 +1,59 @@
+@import url("../section-metadata/section-metadata.css");
+
+.quiz-results {
+  visibility: hidden;
+}
+
+.quiz-results.show {
+  visibility: unset;
+}
+
+.quiz-results .tablet-portrait-up, .quiz-results .tablet-landscape-up,
+.quiz-results .section.desktop-up, .quiz-results .big-desktop-up {
+  display: none;
+}
+
+@media screen and (min-width: 600px) {
+  .quiz-results .tablet-portrait-up {
+    display: block;
+  }
+
+  .quiz-results .phone-only,
+  .quiz-results .tablet-landscape-up,
+  .quiz-results .desktop-up, .quiz-results .big-desktop-up {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 900px) {
+  .quiz-results .tablet-landscape-up {
+    display: block;
+  }
+
+  .quiz-results .tablet-portrait-up,
+  .quiz-results .desktop-up, .quiz-results .big-desktop-up {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .quiz-results .desktop-up {
+    display: block;
+  }
+
+  .quiz-results .tablet-portrait-up, .quiz-results .tablet-landscape-up,
```

Comment:
nit. New line after the comma

Reply:
Commas should be in the appropriate places in CSS now.